conda/exceptions.py

```diff
@@ -617,6 +617,13 @@ def __init__(self, packages, channel_urls=()):
 
             and use the search bar at the top of the page.
             """)
+            from .base.context import context
```

Comment:
Do we need to import this inline? This is a little unfortunate.

Reply:
See precedent in other exceptions. I am not sure why, maybe import times?